cps/web.py

```diff
@@ -1794,49 +1814,51 @@ def unread_books(page):
 @login_required_if_no_ano
 def read_book(book_id, book_format):
     book = db.session.query(db.Books).filter(db.Books.id == book_id).first()
-    if book:
```

Comment:
Whitespace differences because I rewrote the if-statement to be fail-fast (bail out on a failing condition). This let me remove indentation for the positive side of the if-statement.